README.md

```diff
@@ -8,21 +8,219 @@
+
+   **Linux/macOS (Bash)**
+
+   Use the following `curl` command in your terminal:
+
+   ```bash
+   curl --location "https://api.hubapi.com/oauth/v1/token" \
+   --header "Content-Type: application/x-www-form-urlencoded" \
+   --data-urlencode "grant_type=authorization_code" \
+   --data-urlencode "code=<CODE>" \
+   --data-urlencode "redirect_uri=<YOUR_REDIRECT_URI>" \
+   --data-urlencode "client_id=<YOUR_CLIENT_ID>" \
+   --data-urlencode "client_secret=<YOUR_CLIENT_SECRET>"
+   ```
+
+   **Windows (CMD)**
+
+   Use the following `curl` command in Command Prompt:
+
+   ```bash
+   curl --location "https://api.hubapi.com/oauth/v1/token" ^
+   --header "Content-Type: application/x-www-form-urlencoded" ^
+   --data-urlencode "grant_type=authorization_code" ^
+   --data-urlencode "code=<CODE>" ^
+   --data-urlencode "redirect_uri=<YOUR_REDIRECT_URI>" ^
+   --data-urlencode "client_id=<YOUR_CLIENT_ID>" ^
+   --data-urlencode "client_secret=<YOUR_CLIENT_SECRET>"
+   ```
+
+   This command will return the access token and refresh token which are necessary for API calls.
+
+      ```json
+      {
+         "token_type": "bearer",
+         "refresh_token": "<Refresh Token>",
+         "access_token": "<Access Token>",
+         "expires_in": 1800
+      }
+      ```
+
+5. Store the refresh token securely for use in your application.
+
+6. If you are using testing tools (e.g., Postman, Insomnia) or need to manually refresh the token for testing, run the following `curl` command to refresh the access token when it expires, make a POST request to the HubSpot OAuth endpoint.
+Replace the `<YOUR_REFRESH_TOKEN>` ,`<YOUR_CLIENT_ID>` and `<YOUR_CLIENT_SECRET>` with your specific value.
+
+   **Linux/macOS (Bash)**
+
+   Use the following `curl` command in your terminal:
+
+   ```bash
+   curl --request POST \
+   --url https://api.hubapi.com/oauth/v1/token \
+   --header 'content-type: application/x-www-form-urlencoded' \
+   --data 'grant_type=refresh_token&refresh_token=<YOUR_REFRESH_TOKEN>&client_id=<YOUR_CLIENT_ID>&client_secret=<YOUR_CLIENT_SECRET>'
+   ```
+
+   **Windows (CMD)**
+
+   Use the following `curl` command in your command prompt:
+
+   ```bash
+   curl --request POST ^
+   --url https://api.hubapi.com/oauth/v1/token ^
+   --header 'content-type: application/x-www-form-urlencoded' ^
+   --data 'grant_type=refresh_token&refresh_token=<YOUR_REFRESH_TOKEN>&client_id=<YOUR_CLIENT_ID>&client_secret=<YOUR_CLIENT_SECRET>'
+   ```
 
 ## Quickstart
 
-[//]: # (TODO: Add a quickstart guide to demonstrate a basic functionality of the module, including sample code snippets.)
+To use the `Hubspot CRM Associations Schema` connector in your Ballerina application, update the `.bal` file as follows:
+
+### Step 1: Import the module
+
+Import the `hubspot.crm.associations.schema` module and `oauth2` module.
+
+   ```ballerina
+   import ballerina/oauth2;
+   import ballerinax/hubspot.crm.associations.schema as hsAssociationSchema;
```

Comment:
Use lowercase letters for module aliases. (They are treated differently than the variables). Check all the occurrences and update them.

```suggestion
   import ballerinax/hubspot.crm.associations.schema as hsschema;
```